libs/perun/services/src/lib/auth.service.ts

```diff
@@ -46,14 +47,35 @@ export class AuthService {
       sessionStorage.setItem('baLogout', 'true');
       void this.router.navigate(['/service-access'], { queryParamsHandling: 'preserve' });
     } else {
+      this.logoutProcess = true;
       this.oauthService.logOut();
+
+      const postLogoutUrl = this.store.getProperty('oidc_client').oauth_post_logout_redirect_uri;
+      if (!postLogoutUrl) {
+        // redirect to the login page if there is no postLogoutUrl
+        void this.router.navigate(['/login'], { queryParamsHandling: 'preserve' });
+      } else if (this.store.getProperty('proxy_logout')) {
+        // redirect to the logout loading page if postLogoutUrl exist and logout should be handled by proxy
+        void this.router.navigate(['/logout'], { queryParamsHandling: 'preserve' });
+      } else {
+        // directly redirect if postLogoutUrl exist and logout should be handled locally (not by proxy)
+        void this.router.navigate([postLogoutUrl], { queryParamsHandling: 'preserve' });
```

Comment:
I assumed the post logout url is somewhere outside of the web app. But the directly redirect locally case navigates to `instanceUrl/postLogoutUrl`. But maybe this is what was expected ?